index.html

```diff
@@ -6,9 +6,37 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>portfolio</title>
 
+    <!-- Bootstrap icons link -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css"
+    />
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <h1 class="header">"Hello!"</h1>
+    <header class="header">
+      <nav class="navbar">
+        <div class="logo">BZ.</div>
+        <i class="bi bi-list"></i>
+      </nav>
+
+      <section class="hero">
+        <div class="hero__title">
+          <h1>
+            I'm Bahoz<br />
+            Glad to see<br />
+            you!
+          </h1>
+        </div>
+        <p class="hero__text">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and <br />
+          experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.
+        </p>
+        <p class="hero__connect">LET’S CONNECT</p>
+        <img src="/images/social_icons.png" alt="image of social icons" />

```

Comment:
- Please make use of Icons here, not just the compact image you used. You can get Each icon/image of the social icon from the Figma design separately.
- You may decide to use a list to display them then give the list a flex of row. like so.
```html
<ul>
   <li>  img of first icon </li>
   <li> img of second icon </li>
    ....
</ul>
```
- Also do not forget to do the list style to none in CSS so as to remove the bullets. If you are confused in anyway, you can reach me out in slack as `kanu mike chibundu`
- ##### Yours
![Screenshot from 2022-05-25 22-15-12](https://user-images.githubusercontent.com/58771507/170368602-ef91f6e0-db73-447a-9f58-33615899cc1f.png)